superset/connectors/sqla/utils.py

```diff
@@ -86,7 +83,7 @@ def get_physical_table_metadata(
             col.update(
                 {
                     "type": "UNKNOWN",
-                    "generic_type": None,
+                    "type_generic": None,
```

Comment:
this was a bycatch (see line 79/76 above with the correct key) when I worked on adding proper `TypedDict`s for these objects (opening up separate PR for that one, but I thought it worthwhile to fix this error right away).